estate/models/estate_property.py

```diff
@@ -0,0 +1,22 @@
+from odoo import fields, models
+
+
+class EstatePropertyModel(models.Model):
+    _name = "estate_property"
```

Comment:
```suggestion
    _name = "estate.property"
```
By convention, models `_name` are separated with dots.